front/lib/api/assistant/configuration.ts

```diff
@@ -366,15 +366,35 @@ async function fetchWorkspaceAgentConfigurationsWithoutActions(
       });
     default:
       if (typeof agentsGetView === "object" && "agentIds" in agentsGetView) {
-        return AgentConfiguration.findAll({
+        const agents = await AgentConfiguration.findAll({
```

Comment:
It seems a bit tough to not leverage the SQL query, no? Agent configurations can have hundreds or versions 🤔

Reply:
LEt's see the SQL query once you have it. Maybe we can map it to Sequelize.